conf/defaults.ini

```diff
@@ -1038,6 +1038,9 @@ instrumentations_console_enabled = false
 # Should webvitals instrumentation be enabled, only affects Grafana Javascript Agent
 instrumentations_webvitals_enabled = false
 
+# level of internal logging for debugging Grafana Javascript Agent. 0 = OFF, 4 = VERBOSE
```

Comment:
Can only accept 0 or 4? I'd add all the specific values this option accepts. WDYT?

Reply:
Updated in 80bed7a5a35b41efc191feaf22120d659340a3d9